ffmpeg@4.3.1-221: download via aria2 failed

```
Updating 'ffmpeg' (4.3.1-30 -> 4.3.1-221)
Downloading new version
Starting download with aria2 ...
Download: 02/17 09:28:20 [ERROR] CUID#8 - Download aborted. URI=https://github.com/BtbN/FFmpeg-Builds/releases/download/Download: Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://github.com/BtbN/FFmpeg-Builds/releases/download/auDownload: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 754a02|ERR |       0B/s|C:/Users/Mark/scoop/cache/ffmpeg#4.3.1-221#https_github.com_BtbN_FFmpeg-Builds_releases_download_autobuild-2021-02-03-12-34_ffmpeg-n4.3.1-221-gd08bcbffff-win64-gpl-4.3.zip
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 3) Resource was not found
ERROR https://github.com/BtbN/FFmpeg-Builds/releases/download/autobuild-2021-02-03-12-34/ffmpeg-n4.3.1-221-gd08bcbffff-win64-gpl-4.3.zip
    referer=https://github.com/BtbN/FFmpeg-Builds/releases/download/autobuild-2021-02-03-12-34/
    dir=C:\Users\Mark\scoop\cache
    out=ffmpeg#4.3.1-221#https_github.com_BtbN_FFmpeg-Builds_releases_download_autobuild-2021-02-03-12-34_ffmpeg-n4.3.1-221-gd08bcbffff-win64-gpl-4.3.zip

ERROR & 'C:\Users\Mark\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\Mark\scoop\cache\ffmpeg.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/7.1 (Windows NT 10.0; Win64; x64; Core)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=2072 --continue --summary-interval 0
```

--- a/bucket/ffmpeg.json
+++ b/bucket/ffmpeg.json
@@ -5,8 +5,8 @@
     "license": "GPL-3.0-or-later",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/BtbN/FFmpeg-Builds/releases/download/autobuild-2021-02-03-12-34/ffmpeg-n4.3.1-221-gd08bcbffff-win64-gpl-4.3.zip",
-            "hash": "1c838841064fbf84a2be538ec026daa24690e391a1d1e9403baae4d740129503",
+            "url": "https://github.com/BtbN/FFmpeg-Builds/releases/download/autobuild-2021-02-17-12-40/ffmpeg-n4.3.1-221-gd08bcbffff-win64-gpl-4.3.zip",
+            "hash": "f75b827db7b3845df321e3af536c9d21dcf960d68a564dbdb63971d2f72ac927",
             "extract_dir": "ffmpeg-n4.3.1-221-gd08bcbffff-win64-gpl-4.3"
         }
     },

